keel-core/src/main/kotlin/com/netflix/spinnaker/keel/artifacts/DockerArtifact.kt

```diff
@@ -37,5 +36,13 @@ data class DockerArtifact(
 
   @JsonIgnore
   override val statuses: Set<ArtifactStatus> = emptySet()
+
+  override val sortingStrategy: SortingStrategy
```

Comment:
The sorting strategy is auto-selected based on whether the artifact spec is defined in terms of source details or not.